lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -516,8 +516,8 @@ func (d *zfs) CreateVolumeFromCopy(vol Volume, srcVol Volume, copySnapshots bool
 			return err
 		}
 
-		// If using "zfs.clone_copy" delete the snapshot at the end.
-		if (d.config["zfs.clone_copy"] != "" && !shared.IsTrue(d.config["zfs.clone_copy"])) || len(snapshots) > 0 {
+		// If zfs.clone_copy is disabled delete the snapshot at the end.
```

Comment:
@stgraber I just wanted to flag this one for you to check, as the comment doesn't align with the old logic.

Reply:
Ah yeah, it's a bad comment indeed. `clone_copy` is on by default so it's indeed only if it's disabled that we use a snapshot + send/receive and so have to delete the snapshot.